Meadowlark-js/backends/meadowlark-aws-dynamodb-opensearch/test/GenerateReferenceItems.test.ts

```diff
@@ -37,25 +37,26 @@ describe('when collecting foreign key references', () => {
     let result: TransactWriteItem[];
     const parentHash: string = 'ID#09876554tuiolkjasdfasdfwe2w33afss';
     const parentKey: string = 'parentId=asdfasdf';
-    const personNK: string = 'NK#person.personId=b';
-    const info: EntityInfo = newEntityInfo();
+    const parentIdentity = [{ name: 'parentId', value: 'asdfasdf' }];
+    const personNK: DocumentIdentity = [{ name: 'person.personId', value: 'b' }];
+    const info: DocumentInfo = newEntityInfo();
 
     info.projectName = 'Ed-Fi';
     info.entityName = 'Parent';
     info.projectVersion = '3.3.1-b';
     info.foreignKeys.push({
       metaEdType: 'domainEntity',
       metaEdName: 'Person',
-      constraintKey: personNK,
+      documentIdentity: personNK,
       isAssignableFrom: false,
     });
     info.foreignKeys.push({
       metaEdType: 'domainEntity',
       metaEdName: 'EducationOrganization',
-      constraintKey: 'NK#educationOrganization.educationOrganizationId=234',
+      documentIdentity: [{ name: 'educationOrganization.educationOrganizationId', value: '234' }],
```

Comment:
This is an interesting example and helps explain my first comment above. We don't have the same item size constraints as in DynamoDB, so we can be a bit more expansive here.